server/commands.go

```diff
@@ -19,7 +20,7 @@ import (
 )
 
 // add Rollapp commands
-func AddRollappCommands(rootCmd *cobra.Command, defaultNodeHome string, appCreator types.AppCreator, appExport types.AppExporter, addStartFlags types.ModuleInitFlags) {
+func AddRollappCommands(rootCmd *cobra.Command, defaultNodeHome string, appCreator types.AppCreator, appExport rdk_types.AppExporter, addStartFlags types.ModuleInitFlags) {
 	dymintCmd := &cobra.Command{
```

Comment:
does this have usages?

Reply:
Yes, see it: https://github.com/dymensionxyz/rollapp-evm/pull/188/files#diff-3ef291e710a554004d50d61607fe864a27e6b4350f3cb764f2cff8b442c2c213L215